share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -36,6 +38,16 @@ else()
     endfunction()
 endif()
 
+macro(add_rocm_core_dependency)
+    # Optionally add depenency on rocm-core
+    # This mainly empty package exists to allow all of rocm
+    # to be removed in one step by removing rocm-core
+    if(ROCM_DEP_ROCMCORE)
+        rocm_join_if_set(", " CPACK_DEBIAN_PACKAGE_DEPENDS "rocm-core")
```

Comment:
When #58 is merged in, these can be replaced with `rocm_package_add_dependencies("rocm-core")`.

Reply:
Ack.
Do you want me to wait for #58 and update this PR?

Reply:
Nah, #58 still has some work to be done, I'll make the change to it after this is merged in.